ui/src/terraui/esbuild.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path')
+
+require('esbuild').build({
+  loader: { '.jpg': 'file', },
+  bundle: true,
+  format: 'esm',
+  outdir: '/Users/dmohs/b/r/aou/vmlistpage/ui/src/terraui/out',
```

Comment:
Would a build directory in the terraui subdirectory work ok?

Reply:
YES! Wow.
```suggestion
  outdir: 'out',
```